mate-panel/panel-toplevel.c

```diff
@@ -448,7 +448,10 @@ static void panel_toplevel_begin_grab_op(PanelToplevel* toplevel, PanelGrabOpTyp
 	GdkDevice *keyboard;
 #if GTK_CHECK_VERSION (3, 20, 0)
 	GdkSeat *seat;
+	GdkSeatCapabilities capabilities;
 #else
+	GdkDevice *pointer;
```

Comment:
pointer and keyboard are already defined at lines 447 and 448 - should they be removed from there?